lib/teleterm/gateway/config.go

```diff
@@ -69,7 +72,13 @@ func (c *Config) CheckAndSetDefaults() error {
 	}
 
 	if c.LocalAddress == "" {
-		c.LocalAddress = "localhost"
+		// due to SQL Server Management Studio behavior, it can't connect to address like localhost:12345,
+		// we have to use IP, like 127.0.0.1:12345
```

Comment:
```suggestion
		// Due to SQL Server Management Studio behavior, it can't connect to address like localhost:12345,
		// we have to use IP, like 127.0.0.1:12345.
```